### DISCUSSION TITLE
qute-bitwarden can't distinguish username and password fields

### DISCUSSION THREAD

**Discussion by murchu27** [qutebrowser/qutebrowser#5951] [Q&A] [answered]
**Version info**:
qutebrowser v1.14.0
Git commit: 
Backend: QtWebEngine (Chromium 80.0.3987.163)
Qt: 5.15.0

Platform: Manjaro Linux

**Does the bug happen if you start with `--temp-basedir`?**:
Yes

**Description**
The login process for some sites (e.g., Google) is separated into two screens, the first for entering username/email, and the second for password.

Say I'm on Google's login screen, and I run the qute-bitwarden userscript with:

     spawn --userscript qute-bitwarden

The email input field gets filled, no problem. Then I hit "Next" to get to the password input field. I'm hoping that when I run the script again, it will fill my password in the field. However, with the "Show password" checkbox toggled, I can see that it is actually filling my username again.

Is there something I'm missing here? Or is this just not something the script is able to do at the moment?

From my understanding of this userscript, it works by simulating key presses like "{User}\<TAB\>{Password}", so I guess it makes sense that this isn't working, but ideally the script should accommodate for login processes that put the two fields on separate screens, now that such processes are getting more common.

**Comment by The-Compiler** [comment, marked as answer]
It looks like the script has `--username-only` and `--password-only` flags (see `--help`) - I guess those would help here.
